.github/workflows/docs.yml

```
@@ -0,0 +1,85 @@
+  deploy:
+    name: Deploy to GitHub Pages
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    runs-on: ubuntu-latest
+    needs: build
+    if: github.ref == 'refs/heads/main'
+    steps:
+      - name: Deploy to GitHub Pages
+        id: deployment
+        uses: actions/deploy-pages@v4 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)